.github/workflows/build.yml

```diff
@@ -77,12 +77,11 @@ jobs:
       fail-fast: false
       matrix:
         platform:
-          - x86_64
```

Comment:
Why is this removed? Apple have announced that Intel support is going away in future macOS versions, but it isn't dead yet. 😉

Reply:
Oh, lol. GitHub made macos-14+ intel runners only available on Team/Enterprise plans. Never mind. 🙃